parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1188,43 +1179,88 @@ displayRendered outputLoc pp =
         P.indentN 2 pp
       ]
 
-displayDefinitions :: Var v => Ord a1 =>
-  Maybe FilePath
-  -> PPE.PrettyPrintEnvDecl
-  -> Map Reference.Reference (DisplayObject () (DD.Decl v a1))
-  -> Map Reference.Reference (DisplayObject (Type v a1) (Term v a1))
-  -> IO Pretty
-displayDefinitions _outputLoc _ppe types terms | Map.null types && Map.null terms =
-  pure $ P.callout "😶" "No results to display."
+displayDefinitions ::
+  Var v =>
+  Ord a1 =>
+  Maybe FilePath ->
+  PPE.PrettyPrintEnvDecl ->
+  Map Reference.Reference (DisplayObject () (DD.Decl v a1)) ->
+  Map Reference.Reference (DisplayObject (Type v a1) (Term v a1)) ->
+  IO Pretty
+displayDefinitions _outputLoc _ppe types terms
+  | Map.null types && Map.null terms =
+    pure $ P.callout "😶" "No results to display."
 displayDefinitions outputLoc ppe types terms =
   maybe displayOnly scratchAndDisplay outputLoc
   where
-  displayOnly = pure code
-  scratchAndDisplay path = do
-    path' <- canonicalizePath path
-    prependToFile code path'
-    pure (message code path')
-    where
-    prependToFile code path = do
-      existingContents <- do
-        exists <- doesFileExist path
-        if exists then readFile path
-        else pure ""
-      writeFile path . Text.pack . P.toPlain 80 $
-        P.lines [ code, ""
-                , "---- " <> "Anything below this line is ignored by Unison."
-                , "", P.text existingContents ]
-    message code path =
-      P.callout "☝️" $ P.lines [
-        P.wrap $ "I added these definitions to the top of " <> fromString path,
-        "",
-        P.indentN 2 code,
-        "",
-        P.wrap $
-          "You can edit them there, then do" <> makeExample' IP.update <>
-          "to replace the definitions currently in this namespace."
-      ]
-  code = displayDefinitions' ppe types terms
```

Comment:
Are you sure this wasn't changed?

Looks like this used to be pulled out separately and is now inlined for some reason, but I don't see `displayDefinitions'` being edited elsewhere 🤔